---
layout:        post
title:         "Python3 | jinja2 相关模块导入异常"
subtitle:      "ImportError: cannot import name 'json' from itsdangerous"
author:        "Haauleon"
header-img:    "img/in-post/post-python/bg.jpeg"
header-mask:   0.4
catalog:       true
tags:
    - Python
---


### 问题描述
将 python3.6 升级到 python3.8，各种包安装完成后，运行脚本，报错：`ImportError: cannot import name evalcontextfilter, Markup, escape from 'jinja2'`                   

<br>
<br>

### 解决办法